components/buttons/buttons.css

```diff
@@ -0,0 +1,130 @@
+@import url(/stylesheets/default.css);
+
+main header {
+  height: 100%;
+}
+.buttons-link {
+  background-color: var(--extra-color);
+}
```

Comment:
If `.buttons-link ` only changes background color then naming creates a little bit confusing to me. first if it single button then it should be `.button`.

Reply:
This is for the side navigation so that it lights up when we are on that component. so button-link background color changes when we click button link and open button component

Reply:
fine